build.rs

```diff
@@ -18,7 +18,7 @@ fn build_egl(target: &str) {
 
     let mut build = cc::Build::new();
 
-    let data = build_data::EGL;
+    let data = build_data::ANGLE;
```

Comment:
I'm concerned that some of these changes seem to revert the changes from #15 that generated libEGL.dll separates from the rest of angle on windows.

Reply:
Yes, that's a fair concern. I don't know much about what this is meant to do, and I didn't try to investigate further. I see that the script no longer generates `EGL` constant, so not sure how `build.rs` is supposed to work?

Reply:
I didn't realize that build_data was generated, so I didn't update the script. My changes in #15 separated the libEGL parts of the build from the rest of angle, producing a DLL which could be used by Servo. This replicates the effect of the moz.build setup that Gecko uses to produce it as well.

Reply:
I see. So it sounds like #15 needs to be mostly rewritten to affect the `generate_build_data.py` instead of `build_data.rs`